src/test/java/seedu/address/storage/JsonAdaptedItemTest.java

```diff
@@ -30,6 +32,7 @@
             .map(JsonAdaptedTag::new)
             .collect(Collectors.toList());
     private static final String VALID_MAX_QUANTITY = DUCK.getMaxQuantity().toString();
```

Comment:
should be .getMaxQuantity().get().toString()

Reply:
Agree. MaxQuantity is an Optional<Quantity>

Reply:
Fixed in latest commit. Thanks all